src/App.tsx

```diff
@@ -1,10 +1,54 @@
+/* eslint-disable react/jsx-one-expression-per-line */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { getTodos } from './api/todos';
+import ErrorToast from './components/ErrorToast';
+import TodoFooter from './components/TodoFooter';
+import TodoItem from './components/TodoItem';
+import { Todo, TodoStatus } from './types/Todo';
 
-const USER_ID = 0;
+const USER_ID = 11033;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
```

Comment:
the variable `todos` can be calculated based on `todosContainer` and `selectedStatus` by using `useMemo` hook